elements/pfe-navigation/README.md

```diff
@@ -0,0 +1,65 @@
+# PatternFly Element | Navigation Element
+
+## Navigation element
+
+This element will give you a top-level navigation element (not for use as subnavigation).
+
+### Slots
+
+- `skip`: This slot contains all the content for the skip navigation. Best practice is to use an unordered list with link tags that link to anchors on the page relevant to the users. These anchors can include things like: "main content", "registration form", "contact us", etc. Currently this is only available to screen-readers.
+- `logo`: This slot should contain the site's main logo.
+- `search`: This slot, when assigned to a navigation-item component, renders a dropdown in the navigation for the search functionality. Inside the tray for the slot, we recommend tagging the search form or search functionality that includes the input and submit button with the attribute `pfe-navigation--mobile-search`. The element inside the search slot that is tagged with this attribute will be copied into the mobile menu and appear above the accordion for the main navigation.
+- `language`: This slot, when assigned to a navigation-item component, renders the dropdown for the user to select the site language.
+- `mobile-language`: A simple link to a landing page containing language switching information.  Example: `<a href="/url-to-language-page" slot="mobile-language" pfe-icon="user" hidden>English</a>`.
+- `login`: This slot, when assigned to a navigation-item component, renders the dropdown for the user to log into the site.
+- `mobile-login`: A simple link to a landing page containing language switching information.  Example: `<a href="/login" slot="mobile-login" pfe-icon="user" hidden>Login/Register</a>`.
+- `site-switcher`: This slot, when assigned to a navigation-item component, renders the dropdown for the site switcher, allowing the user to navigate an ecosystem of websites.
+
+### Attributes
+
+- `pfe-sticky`: Allows the navigation to stick to the top of the page when the user scrolls down.
+- `pfe-close-on-click`: Currently only supports "external"; this means if a user clicks outside the component, the navigation items will close.
+
+## Navigation item
+
+A navigation item is a single dropdown element; the navigation is composed of sets of navigation-items.
+
+### Slots
+
+- `trigger`: Accepts a link tag with a fallback URL (used if the JS doesn't upgrade). This slot should be applied to an h-level tag. Example: `<h3 slot="trigger"><a href="http://www.redhat.com">Home</a></h3>`.
+- `tray`: Accepts any light DOM content to be rendered inside the tray. Example: `<div slot="tray" hidden></div>`.  We recommend using the `pfe-navigation-item__tray--container` class on the wrapper element inside the tray. This will give the tray a max-width that matches the navigation tray.
+
+### Attributes
+
+- `pfe-icon`: Currently available icons are "bento", "user", "globe", "search", and "menu".  Adding this attribute renders that icon above the trigger.
+
```

Comment:
Maybe it's overkill, but should we include the `hidden` attribute here where we explain that it should always be attached to the tray slot?

Reply:
+1 updating now